src/cms/page-builder-elements-buttons.md

```diff
@@ -80,6 +80,8 @@ _Buttons container toolbox_
 
    - **Product** - Enter a product name (partial or full) or SKU, then choose the product name in the list.
 
+      Please note that the products are displayed in the dropdown with the respect of _Show out of stock products_ settings value. For Multi Source merchants using Inventory Management, the products list is limited by the source assigned to the default website only.
```

Comment:
We use the note format rather than saying "Please note". Also I think we should link to Inventory management if we mention it here.

```suggestion
      {:.bs-callout-info}
      The products are displayed in the list according to the _Show out of stock products_ settings. For Multi Source merchants using [Inventory Management]({% link catalog/inventory-management.md %}), the products list is limited by the source assigned to the default website only.
```